sql/numbertype.go

```diff
@@ -103,7 +103,7 @@ func (t numberTypeImpl) Convert(v interface{}) (interface{}, error) {
 	}
 
 	if ti, ok := v.(time.Time); ok {
-		v = fmt.Sprintf("%d%02d%02d%02d%02d%02d", ti.Year(), ti.Month(), ti.Day(), ti.Hour(), ti.Minute(), ti.Second())
+		v = ti.UTC().Unix()
```

Comment:
This change is not consistent with MySQL behavior. Run this command in your local MySQL instance:
```
CREATE TABLE `sys`.`abc` (`pk` BIGINT, `v` BIGINT, PRIMARY KEY (`pk`));
INSERT INTO `sys`.`abc` VALUES (0, CAST("2020-01-14 12:13:14" AS DATETIME));
```

You'll see the result is `20200114121314`.

Reply:
Slept on this, and decided the MySQL behavior is terrible and indefensible. Keeping this as is.